README.md

```diff
@@ -67,6 +67,36 @@ Winit provides the following features, which can be enabled in your `Cargo.toml`
 * `wayland` (enabled by default): On Unix platform, compiles with the Wayland backend
 * `mint`: Enables mint (math interoperability standard types) conversions.
 
+## MSRV Policy
+
+The Minimum Supported Rust Version (MSRV) of this crate is **1.65**. Changes to 
+the MSRV will be accompanied by a minor version bump. 
+
+As a **tentative** policy, the upper bound of the MSRV is given by the following
+formula:
+
+```
+min(sid, stable - 3)
+```
+
+Where `sid` is the current version of `rustc` provided by [Debian Sid], and
+`stable` is the latest stable version of Rust. This bound may be broken in the
+event of a major ecosystem shift or a security vulnerability.
+
+[Debian Sid]: https://packages.debian.org/sid/rustc
+
+The exception to this is for the Android platform, where a higher Rust version 
+must be used for certain Android features. In this case, the MSRV will be 
+capped at the latest stable version of Rust minus three. This inconsistency is 
+not reflected in Cargo metadata, as it is not powerful enough to expose this 
+restriction.
+
+All crates in the [`rust-windowing`] and [`rust-mobile`] organizations have the 
```

Comment:
I don't really think we should be speaking for all rust-mobile crates here.

I consider rust-mobile crates to be independently maintained, and different projects are free to implement their own MSRV policy.

When I've invited devs to move crates to rust-mobile I have specifically said that they remain responsible for their crate.

This isn't to say that rust-mobile maintainers can't choose to harmonize things like their MSRV policy with other crates if that makes sense, but this isn't really the place to impose this.